src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Members.cs

```diff
@@ -59,7 +59,13 @@ public override void VisitField(IFieldSymbol symbol)
                 AddPunctuation(SyntaxKind.DotToken);
             }
 
-            if (symbol.ContainingType.TypeKind == TypeKind.Enum)
+            if (symbol.AssociatedSymbol is IPropertySymbol associatedProperty)
```

Comment:
>if (symbol.AssociatedSymbol is IPropertySymbol associatedProperty) [](http://example.com/codeflow?start=12&length=66)

Is this going to affect how VB symbols are displayed? We probably don't want that because backing fields in VB have speak-able, user-referenceable, names. We might want to do this substitution only for C# symbols.

Reply:
I am expecting this change to only affect C# symbols due to being in C# layer. I will look for, or add, a VB test to demonstrate that an appropriate display of the backing field is used there.

Reply:
> I am expecting this change to only affect C# symbols due to being in C# layer.

I do not think this is the case. This layer can be invoked on VB symbols. It operates on common interfaces. I am sure we have a bunch of tests checking C# display for VB symbols and checking VB display for C# symbols.